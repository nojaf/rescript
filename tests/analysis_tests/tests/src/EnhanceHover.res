// ^dv+
let xxxxxxx = [1, 2, 3]
//  ^hov

let xxxxxxxx = [1, 2, 3]
//             ^hov

let xxxxxxxxx = [1, 2, 3]
//               ^hov
