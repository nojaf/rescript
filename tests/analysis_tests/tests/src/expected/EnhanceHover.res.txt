
Hover src/EnhanceHover.res 1:4
{"contents": {"kind": "markdown", "value": "```rescript\narray<int>\n```"}}

Hover src/EnhanceHover.res 4:15
{"contents": {"kind": "markdown", "value": "```rescript\narray<int>\n```"}}

Hover src/EnhanceHover.res 7:17
{"contents": {"kind": "markdown", "value": "```rescript\nint\n```"}}

